e2e_test/batch/functions/encdec.slt.part

```diff
@@ -33,6 +33,11 @@ SELECT decode('MTIzAAE=', 'base64');
 ----
 \x3132330001
 
+query T
+SELECT decode('MTIz/A+=', 'base64');
+----
+\x313233fc0f
+
```

Comment:
I think this might break some users' data. We should leave a comment re breaking change.

Reply:
I think it won't because the bug is in the decoding base64 and encoding base64 is correct.

Before this PR:
- Invalid base64 string containing `*` can be successfully decoded. This can happen only if the user provided base64 string is invalid.
- Valid base64 string containing `+` cannot be successfully decoded

After this PR:
- Invalid base64 string containing `*` cannot be successfully decoded
- Valid base64 string containing `+` can be successfully decoded